exasol_transformers_extension/utils/bucketfs_model_specification.py

```diff
@@ -8,13 +8,26 @@
 
 class BucketFSModelSpecification(ModelSpecification):
     """
-    Class describing a model with additional information about
-    the bucketFS connection and the subdir in the bucketfs the model can be found at.
+    Describes a model with additional information about the BucketFS
+    connection and the subdir in the BucketFS the model should be uploaded to
+    or can be found at after uploading.
     """
 
     def __init__(
         self, model_name: str, task_type: str, bucketfs_conn_name: str, sub_dir: Path
     ):
+        """
+        model_name:
+            Name of the model. This is the same name as it's seen on the Haggingface
+            model card, for example 'cross-encoder/nli-deberta-base'.
+        task_type:
```

Comment:
this is not quite correct. the TE uses the udf-names as input task names, not the transformers one

Reply:
I copied this sentence from somewhere else. But I am happy to change the docstring. 

Can you please propose a sentence?

Reply:
Many thanks for assisting me to fix the docstring. Please see latest push for an updated version.